includes/multi-currency/Settings.php

```diff
@@ -88,11 +88,10 @@ public function get_settings( $current_section = '' ) {
 				[
 					[
 						'title' => __( 'Enabled currencies', 'woocommerce-payments' ),
-						// TODO: Learn more link needed.
 						'desc'  => sprintf(
 							/* translators: %s: url to documentation. */
-							__( 'Accept payments in multiple currencies. Prices are converted based on exchange rates and rounding rules, and include conversion fees. <a href="%s">Learn more</a>', 'woocommerce-payments' ),
-							''
+							__( 'Accept payments in multiple currencies. Prices are converted based on exchange rates and rounding rules, and include conversion fees. <a href="%s" target="blank">Learn more</a>', 'woocommerce-payments' ),
+							'https://docs.woocommerce.com/document/payments/currencies/multi-currency-setup/'
```

Comment:
Perhaps we should add the link to the docs as a `const` at the top of the file so that we can update it easily in one place if needed in future?

Reply:
Thank for the review Daniel, added the change in 62518a38.

Reply:
Looks good, thanks for making that change! 

It looks like Psalm is complaining about something here: https://github.com/Automattic/woocommerce-payments/pull/2554/checks?check_run_id=3201195456

Once that is fixed, this looks good to merge to me! 👍

Reply:
🤦 I fixed the merge conflict on GH and left an empty line. Just fixed it up. Will merge after the builds are done.